aten/src/ATen/native/quantized/cpu/fbgemm_utils.h

```diff
@@ -2,6 +2,7 @@
 
 #include <c10/core/QScheme.h>
 #include <ATen/Tensor.h>
+#include <ATen/cpp_custom_type_hack.h>
```

Comment:
FYI: we have ongoing PRs to remove cpp_custom_type_hack: https://github.com/pytorch/pytorch/pull/34140